docs/develop/api-reference/dictionary-collections.md

```diff
@@ -0,0 +1,201 @@
+---
+sidebar_position: 1
+sidebar_label: Dictionary collections
+title: Dictionary collection API reference
+description: Learn how to interact with the API for dictionary collection data types in Momento Serverless Cache.
+slug: /develop/api-reference/collections/dictionary
+---
+
+# Dictionary collection API reference
+
+## Dictionary methods
+This page details the Momento API methods for [dictionary collections](./../datatypes.md#dictionary-collections).
+
+### DictionaryFetch
+
+Gets a dictionary item from a cache.
+
+| Name            | Type   | Description                                   |
+| --------------- | ------ | --------------------------------------------- |
+| cacheName       | String | Name of the cache.                            |
+| dictionaryName  | String | The name of the dictionary to be retreived. |
+
+<details>
+  <summary>Method response object</summary>
+
+The response object for DictionaryFetch returns three possible options, a cache hit, miss, or an error.
+
+* Cache hit
+    - `valueDictionaryBytesBytes()`: Map<Bytes, Bytes>
+    - `valueDictionaryStringString()`: Map<String, String>
+    - `valueDictionaryStringBytes()`: Map<String, Bytes>
+    - `valueDictionaryBytesString()`: Map<Bytes, String>
+    - `toString()`: string - displays the key/value pairs, truncated.
+* Cache miss
+* Cache error
+
+See [response objects](./response-objects.md) for specific information.
+
+</details>
+
+### DictionaryGetField
+Get one field from a dictionary item in the cache.
```

Comment:
Same comment here, to me the word "item" after "dictionary" is superfluous and possibly confusing.